.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules
+coverage
+.DS_Store
```

Comment:
I'm fine with this here.  I don't do it globally either and I bet a lot of people don't